src/vue/pages/Assets.vue

```diff
@@ -121,10 +121,22 @@ export default {
 </script>
 
 <style lang="scss">
+@import '~@scss/variables';
+
 .assets-page__btn-icon {
   display: flex;
   font-size: 1.8rem;
   margin-right: 0.5rem;
   margin-top: -0.4rem;
 }
+
+.assets-page__top-bar-filter {
+  max-width: 20rem;
+}
+
+@media (max-width: $sidebar-hide-bp) {
```

Comment:
.assets-page__create-btn {
@include respond-to-custom($sidebar-hide-bp) {
margin-top: 0.4rem;
}
}

Reply:
changed

Reply:
changed